lib/matplotlib/projections/polar.py

```diff
@@ -561,8 +560,7 @@ def set_rgrids(self, radii, labels=None, angle=None, fmt=None,
         if angle is None:
             angle = self.get_rlabel_position()
         self.set_rlabel_position(angle)
-        for t in self.yaxis.get_ticklabels():
-            t.update(kwargs)
+        self.yaxis.set_tick_params(which='major', **kwargs)
```

Comment:
Unfortunately, you're right to be suspicious. There's a loop through the input to check that the keys are valid for ticks, and doesn't necessarily allow all possible `Line2D` arguments.